### DISCUSSION TITLE
Chia blockchain redowlaning every time I update the Client.

### DISCUSSION THREAD

**Discussion by dmc1648** [Chia-Network/chia-blockchain#17328] [Support]
The title say's it all.  For the last 3 Client update's the blockchain that was stored on my computer was wiped and re-downloaded.  any reason why?

TIA for any info.

**Comment by BrandtH22** [comment]
Hey @dmc1648 , as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia). 

As for the blockchain db being overwritten, the only times I have seen something like that are either:
- additional -shm and -wal files in teh blockchain db directory `~\.chia\mainnet\db\` , these files have the same name as the blockchain db but end in their respective file types and are used as temporary files for the database. With chia fully stopped you are safe to delete these files.

- manual updates to the config file parameter for the blockchain db location. If these changes are not formatted properly it can lead to odd issues like the db being overwritten or new dbs being created. If you have never updated the config file, it would be a good idea to have a new one created:

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run `chia init`
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
